android/src/main/kotlin/dev/rexios/polar/PolarPlugin.kt

```diff
@@ -153,6 +159,13 @@ class PolarPlugin :
         }
     }
 
+    private fun initSdk() {
+        if (wrapperInternal == null) {
+            wrapperInternal = PolarWrapper(context)
+        }
+        wrapper.addCallback(polarCallback)
+    }
+
     private val searchHandler =
```

Comment:
Since the SDK is not guaranteed to be initialized on startup, we need to initialize it here like in the iOS code